src/index.html

```diff
@@ -17,16 +17,37 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
```

Comment:
The `action` attribute for this form is set to `#`. If this form is supposed to perform a specific action, consider setting the correct URL or endpoint.